### PR TITLE
DDF-6061 Upgrade Mockito, Pac4j, Jackson

### PR DESCRIPTION
**Pull request by SmithJosh**
This is the counterpart of these DDF PRs:
https://github.com/codice/ddf/pull/6062
https://github.com/codice/ddf/pull/6070

It upgrades dependencies to match the version in DDF.

### CODE REVIEW

File: ui-backend/catalog-ui-oauth/pom.xml [hunk 1]
```diff
@@ -54,12 +54,12 @@
         <dependency>
             <groupId>com.nimbusds</groupId>
             <artifactId>nimbus-jose-jwt</artifactId>
-            <version>8.2</version>
+            <version>${nimbus.version}</version>
```
Comment (bakejeyner) [inline ui-backend/catalog-ui-oauth/pom.xml:57]: :bow:

**Review by bakejeyner — APPROVED** [review]

**Review by blen-desta — APPROVED** [review]

**Comment by blen-desta** [comment]
Hero'd successfully with https://github.com/codice/ddf/pull/6070 changes

**Review by parret — APPROVED** [review]

**Comment by jlcsmith** [comment]
@SmithJosh  - DDF has been released so this should be unblocked.  Please update the dependency to DDF 2.25.1.

**Comment by vinamartin** [comment]
Merged in https://github.com/codice/ddf-ui/pull/282

**Pull request closed without merging**